### PR TITLE
Payments / Sending bitcoin intro copy changes

### PR DESCRIPTION
**Pull request by Bosch-0**
Some changes to the top section copy of the [payments/sending](https://bitcoin.design/guide/payments/send/) section of the guide. I am planning on going through this page one section at a time to make the PR review process more manageable. Let me know if reviewers prefer this approach or would rather I go over the whole section in one big PR. 

**Changes made**

- Re-worded the content (most notable the opening paragraph) to put more of a emphasis on _designing send flows_ rather than having it sound like a general _sending bitcoin_ overview. With the context of the guide in mind this makes more sense. A few areas of this section read like an explanation of sending bitcoin without the context of _designing_ sending bitcoin flows like it should. 
- Broke the the list up into must-haves and optional, advanced extras. Added several features, mostly to the advanced section. Content will need to be added in the later sections covering these.

### CODE REVIEW

File: guide/payments/send.md [hunk 1]
```diff
@@ -37,16 +37,25 @@ layout = "full-width"
...
-- **Fee settings** — Prioritize fast confirmation or low cost (optional)
+- **Amount** - The amount of bitcoin the user is sending.
+- **Receiver address** - The Bitcoin Address the user is sending their bitcoin to.
+- **Network fee** - Display to the user the current Bitcoin network fee they will be paying for this send.
 
-You do not need to follow the order below. Feel free to tailor the configuration's order for the payment to what best serves your users. For example, you may make users set the amount before they enter the address.
+Some optional, more complex, features you may include:
+
+- **Description** - Let the user add a description detailing what this send is for. 
+- **Coin selection** - Which coins/inputs, also called UTXOs, to use in this send.
```
Comment (johnsBeharry) [inline guide/payments/send.md:51]: there is now a sub page for coin selection. lets see how we can work this in.

Comment (Bosch-0) [reply guide/payments/send.md:51]: I think it's still worth adding here and including some details later in the page whilst linking to the coin selection page for the deep dive.

**Comment by GBKS** [comment]
So it looks like you rewrote the whole text. It's fine to propose that, I am just curious why you felt like the whole thing needed to be changed. And do you expect to rewrite the rest of the page altogether as well?

This is a more general question. When we feel like a lot about a page needs to change, should we go straight into a PR or first create an issue to lay out the rationale and see whether others agree?

**Comment by Bosch-0** [comment]
It didn't read like it was ['speaking to the reader'](https://bitcoin.design/guide/contribute/content-guidelines/#speak-to-the-reader). It read more like a straight explanation of what these things are. We should be doing that alongside saying why these elements are useful / relevant to their users.

**Comment by johnsBeharry** [comment]
> put more of a emphasis on designing send flows rather than having it sound like a general sending bitcoin overview

- Should both be included in the guide chapter?
- Or should it solely be "flow" based?
- What is the differentiation of the chapter and case studies?

> A few areas of this section read like an explanation of sending bitcoin without the context of designing sending bitcoin flows like it should

- Could you give some examples so we can reference in this discussion?

**Comment by Bosch-0** [comment]
In https://github.com/BitcoinDesign/Guide/pull/341 I made that section more action driven rather than just explaining that an address needs to be shared with the recipient. We should talk in the context of 'your user should do x' rather than 'this is done by x.'

**Comment by GBKS** [comment]
If I understand correctly, @johnsBeharry question is whether this page should only focus on "how it works" and not "how to design for it".

The "Private key management" pages, for example, only describe "how it works". Make sense, as the different schemes can be implemented in various ways. Then the case study pages (which were extracted from the PKM section) do a good job at pulling together a use case, scheme, and "how to design for it" into a single UI/UX package.

Conor took a different route with the "Onboarding" section which is built around a specific product configuration (daily spending, mobile, automatic cloud backup). Also cool, as there isn't as much technical underpinning, but it's more focused on the user experience. Onboarding for other products configurations can also be found in the case studies (like multisig in the Savings account case study).

For sending bitcoin, I could see separating "how it works" and "how to design for it" be helpful, as there are huge differences between different types of implementations. Muun manages to hide a ton of stuff from you, while other wallets make you go through fee and coin selection, or you might need multisig signing, etc. If the one implementation you choose to show on this page leans towards Muun, then you leave out a lot. If it goes too far the other way, then it probably becomes a beast. The page is currently more focused on "how it works" with light UI examples to illustrate each step. Maybe that's the way to go and we add case studies to showcase different implementations (Muun-like, manual, multisig, batch, PayJoin...)?

Just my rambling 2 cents. Feel free to ignore.

**Comment by johnsBeharry** [comment]
Yes @GBKS thats better articulated. Any thoughts on this @Bosch-0 ?

File: guide/payments/send.md [hunk 2]
```diff
@@ -37,16 +37,25 @@ layout = "full-width"
 
 # Sending bitcoin
 
-Sending bitcoin can be a very straightforward or complex flow in a Bitcoin application. People may be sending bitcoin to a known contact, moving it between their wallets on different devices, or making a purchase through a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
+Depending on the target audience of your bitcoin application, designing send flows can range in complexity. An application for new users may focus on the simple flow of entering an amount, address and clicking send. An application for more advanced users may include things like coin selection and custom fee setting as extra options.
```
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/payments/send.md:40]: How about `users who want more control`?

**Comment by Bosch-0** [comment]
Going to close this for now as I imagine this chapters content will change significantly if we are assuming LN going forward.

**Comment by chikta123** [comment]
rGoogle Tag Manager makes it easy for marketers to add and update website and mobile app tags — including conversion tracking, site analytics, remarketing, and more — with just a few clicks. All for free. Marketers get more flexibility to control tags and webmasters get more time to focus on other tasks. Learn more at g.co/tagmanager.© 2018 Google LLC. All rights reserved. Google and the Google logo are trademarks of Google LLC. All other company and product names may be trademarks of the respective companies with rGoogle Tag Manager makes it easy for marketers to add and update website and mobile app tags — including conversion tracking, site analytics, remarketing, and more — with just a few clicks. All for free. Marketers get more flexibility to control tags and webmasters get more time to focus on other tasks. Learn more at g.co/tagmanager.© 2018 Google LLC. All rights reserved. Google and the Google logo are trademarks of Google LLC. All other company and product names may be trademarks of the respective companies with

**Comment by chikta123** [comment]
Join Amazon Music Unlimited Family Plan 
Up to 6 family members can access Amazon Music Unlimited per plan. 

https://a.co/gDsPfsM

**Comment by chikta123** [comment]
collection:web AND ( Google play archive Marketpayout )
www.bitcoin.com

**Pull request closed without merging**